### PR TITLE
Fix readme command to add repository name

### PR DESCRIPTION
**Pull request by SamirBoulil**
Add the repository name to the tag command in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Copy the `config.yml.dist` file and rename it to `config.yml` to add your github
 
 ## Release a new version
 
-    app/console michel:tag 1.7
+    app/console michel:tag 1.7 akeneo
```
Comment (juliensnz) [inline README.md]: It could be better to have another example. I fear that if you put this default value, it could cause some problems

Comment (SamirBoulil) [inline README.md]: I get it, is <repository_name> ok for you ?

Comment (SamirBoulil) [inline README.md]: or even <branch_name> <repository_name> ?

Comment (pierallard) [inline README.md]: <organization_name> in fact, repository_name = pim-community-dev e.g.

**Pull request merged**